packages/react/src/components/header/components/headerActionBar/HeaderActionBarNavigationMenu.tsx

```diff
@@ -17,44 +178,218 @@ type HeaderActionBarNavigationMenuProps = {
+              frontPageLabel={frontPageLabel}
+              titleHref={titleHref}
+              onClick={goBack}
+              openFrontPageLinksAriaLabel={openFrontPageLinksAriaLabel}
+            />
+          )}
+          <ActiveDropdownLink
+            id={!isRenderingDeepestMenu && !isAnimating ? currentActiveLinkId : undefined}
+            link={!isRenderingDeepestMenu ? currentlyActiveMainLink : previousDropdownLink}
+            frontPageLabel={frontPageLabel}
+            titleHref={titleHref}
+            onLinkClick={handleLinkClick}
+          />
+          <MenuLinks links={menuLinks} onDropdownButtonClick={goDeeper} onLinkClick={handleLinkClick} />
+        </NavigationSection>
+        {/* Next links. Rendered at the deepest level. */}
+        {!openingLink && (
+          <NavigationSection
+            universalLinks={universalLinks}
+            aria-hidden={!isRenderingDeepestMenu}
+            className={isAnimating || isRenderingDeepestMenu ? styles.visible : styles.hidden}
+          >
+            <PreviousDropdownLink
+              link={previousDropdownLink}
+              frontPageLabel={frontPageLabel}
+              titleHref={titleHref}
+              onClick={goBack}
+              openFrontPageLinksAriaLabel={openFrontPageLinksAriaLabel}
+            />
+            <ActiveDropdownLink
+              id={isRenderingDeepestMenu ? currentActiveLinkId : undefined}
+              link={currentlyActiveMainLink}
+              frontPageLabel={frontPageLabel}
+              titleHref={titleHref}
+              onLinkClick={handleLinkClick}
+            />
+            <MenuLinks links={menuLinks} onDropdownButtonClick={goDeeper} onLinkClick={handleLinkClick} />
+          </NavigationSection>
+        )}
+        {/* Render the menu animating into view for better UX. */}
+        {openingLink && typeof openingLink !== 'string' && (
+          <NavigationSection
+            universalLinks={universalLinks}
+            aria-hidden={!openingLink}
+            className={isAnimating ? styles.visible : styles.hidden}
+          >
+            <PreviousDropdownLink
+              link={currentlyActiveMainLink}
+              frontPageLabel={frontPageLabel}
+              titleHref={titleHref}
+              onClick={goBack}
+              openFrontPageLinksAriaLabel={openFrontPageLinksAriaLabel}
+            />
+            <ActiveDropdownLink
+              link={openingLink}
+              frontPageLabel={frontPageLabel}
+              titleHref={titleHref}
+              onLinkClick={handleLinkClick}
+            />
+            <MenuLinks
+              links={openingLink.props.dropdownLinks}
+              onDropdownButtonClick={goDeeper}
+              onLinkClick={handleLinkClick}
+            />
+          </NavigationSection>
+        )}
+      </div>
       <HeaderActionBarLogo
         logoProps={{
           ...logoProps,
```

Comment:
The logo's focus should be more compact, here are the [designs.](https://www.figma.com/file/Kxwg3R0zNRHj55nQqFu6VS/HDS-UI-Kit-3.0?type=design&node-id=287-5996&mode=design&t=VpbSKzxDpHZGDZJq-0)

Reply:
https://github.com/City-of-Helsinki/helsinki-design-system/pull/1120/commits/6b990a5e1d5e78088ee6182fa71b19d66ce592e4

Reply:
This looks still the same to me. Apparently my design link isn't working anymore. Anyway I think the focus should be the same as in Header ActionBar logo.